src/components/Categories/Caregories.jsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState, useEffect } from "react";
+import PropTypes from 'prop-types';
+import { NavLink, useLocation } from "react-router-dom";
+import { useSelector } from "react-redux";
+import { categories } from "../../modules/constants/constants";
+import { selectorGenre } from "../../modules/selectors/index";
+
+import "./style.scss";
+
+const Categories = ({ handleMovieListView, movieListView }) => {
+  const [activeLink, setActiveLink] = useState("Trending");
+  const arrResult = useSelector(state => selectorGenre(state));
```

Comment:
let's check
from my point of view it should works so: 
`const arrResult = useSelector(selectorGenre);`

Reply:
and plz check all cases using selectors